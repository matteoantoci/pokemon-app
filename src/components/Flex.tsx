import React from 'react'
import styled from 'styled-components'

const Wrapper = styled.div`
  display: flex;
`

export const Flex: React.FC = ({ children, ...props }) => <Wrapper {...props}>{children}</Wrapper>
