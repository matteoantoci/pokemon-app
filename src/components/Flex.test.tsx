import React from 'react'
import renderer from 'react-test-renderer'
import { Flex } from './Flex'

describe('Flex', () => {
  it('renders correctly', () => {
    const component = renderer.create(<Flex>Contents</Flex>)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
