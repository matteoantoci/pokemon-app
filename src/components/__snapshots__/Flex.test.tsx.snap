// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Flex renders correctly 1`] = `
<div
  className="sc-bdVaJa kZAgjJ"
>
  Contents
</div>
`;
